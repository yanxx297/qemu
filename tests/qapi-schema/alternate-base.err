tests/qapi-schema/alternate-base.json:4: Alternate 'Alt' must not have a base
