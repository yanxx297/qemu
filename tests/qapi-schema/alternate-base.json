# FIXME: we should reject anonymous union with base type
{ 'type': 'Base',
  'data': { 'string': 'str' } }
{ 'union': 'MyUnion',
  'base': 'Base',
  'discriminator': {},
  'data': { 'number': 'int' } }
