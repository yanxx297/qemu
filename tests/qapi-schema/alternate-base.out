[OrderedDict([('type', 'Base'), ('data', OrderedDict([('string', 'str')]))]),
 OrderedDict([('union', 'MyUnion'), ('base', 'Base'), ('discriminator', OrderedDict()), ('data', OrderedDict([('number', 'int')]))])]
[{'enum_name': 'MyUnionKind', 'enum_values': None}]
[OrderedDict([('type', 'Base'), ('data', OrderedDict([('string', 'str')]))])]
